tasks/elasticsearch-Debian.yml

```diff
@@ -26,3 +26,7 @@
   apt: deb=/tmp/elasticsearch-{{ es_version }}.deb
   when: not es_use_repository
   register: elasticsearch_install_from_package
+
+# ansible uri module requires httplib2
+- name: pip httplib2
+  pip: name=httplib2 extra_args="--user"
```

Comment:
hopefully we can remove this all together and only support 2.1+
if not, i'll swap in python-httplib2 thanks